profiles/sam/terraform/sam/variables.tf.ctmpl

```diff
@@ -0,0 +1,203 @@
+#
+# Profile Vars
+#
```

Comment:
Anything here that isn't used in the other files can go. I also have an idea for reducing the boilerplate further if we want: https://github.com/broadinstitute/dsp-k8s-deploy/pull/17#discussion_r292460611